src/pykoala/corrections/sky.py

```diff
@@ -826,10 +827,10 @@ def __init__(self, dc, bckgr_estimator='mad', bckgr_params=None,
         cont_estimator_args : dict, optional
             Arguments for the continuum estimator. Default is None.
         """
-        self.vprint("Creating SkyModel from input Data Container")
+        vprint(self,"Creating SkyModel from input Data Container")
```

Comment:
Why do you need to pass `self` as an argument?

Reply:
As stated above: 

`vprint` is an unbound function, but in class `SkyFromObject` was used as an attribute. This was causing an issue when running `py.test` and showed a warning in `pylance`. While the execution in VS code was not showing issues, the function should not be run as attribute in this case anyway. For an attribute, it makes sense to have the original line, but when using an external function (defined outside `src/pykoala/corrections/sky.py`)